test/answer.e2e-spec.ts

```diff
@@ -148,61 +149,83 @@ describe('Answers Module', () => {
         expect(respond.status).toBe(201);
         expect(respond.body.data.id).toBeDefined();
         questionId.push(respond.body.data.id);
+        return respond.body.data.id;
       }
-      await createQuestion(
-        '我这个哥德巴赫猜想的证明对吗？',
-        '哥德巴赫猜想又名1+1=2，而显然1+1=2是成立的，所以哥德巴赫猜想是成立的。',
-      );
-    });
 
+      await Promise.all([
+        createQuestion(
+          '我这个哥德巴赫猜想的证明对吗？',
+          '哥德巴赫猜想又名1+1=2，而显然1+1=2是成立的，所以哥德巴赫猜想是成立的。',
+        ),
+        createQuestion('求助', '给指导老师分配了任务，老师不干活怎么办？'),
+        createQuestion('求助', '给指导老师分配了任务，老师不干活怎么办？'),
+        createQuestion('不懂就问', '忘记给指导老师分配任务了怎么办'),
+        createQuestion('小创求捞', '副教授职称，靠谱不鸽，求本科生带飞'),
+      ]);
+    });
     it('should create an auxiliary user', async () => {
       [auxUserId, auxAccessToken] = await createAuxiliaryUser();
     });
   });
-
-  // describe('answer question', () => {
-  it('should create some answers', async () => {
-    const testQuestionId = questionId[0];
-    async function createAnswer(content: string) {
+  describe('answer question', () => {
+    it('should create some answers', async () => {
+      // const testQuestionId = questionId[0];
+      async function createAnswer(questionId: number, content: string) {
+        const respond = await request(app.getHttpServer())
+          .post(`/questions/${questionId}/answers`)
+          .set('Authorization', `Bearer ${auxAccessToken}`)
+          .send({ content });
+        expect(respond.body.message).toBe('Answer created successfully.');
+        expect(respond.body.code).toBe(200);
+        expect(respond.status).toBe(201);
```

Comment:
the status and body.code should be the same. alter your body.code to 201, if the action is POST